modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -29,7 +29,7 @@ spec:
   profile: <2>
   - name: "profile1" <3>
     interface: "" <4>
-    ptp4lOpts: "-s -2" <5>
+    ptp4lOpts: "-2" <5>
```

Comment:
for  logs to generate and used by he ptp  events, you  need summary interval
  ptp4lOpts: "-2 --summary_interval -4"
  phc2sysOpts: "-a -r -m -n 24 -N 8 -R 16"